pyproject.toml

```
@@ -96,16 +96,19 @@ test = [
     "boltons",
     "scanpy",
     "httpx", # For data downloading
-    "dask[array,distributed]>=2022.09.2,<2024.8.0",
+    "dask[distributed]>=2022.09.2",
```

The constraint is already there through `anndata[dask]`, no need to add it again.

```suggestion
    "dask[distributed]",
```

---

Are `distributed` and `array` linked?

---

i.e.I think you can install different versions of each?  Just tested this.

---

Hmm, no `dask[array]` is _not_ a separate package, ok.

---

Or rather, `distributed` is a separate package, but hooks back to `dask`

---

Hence the confusion

---

`dask[distributed]>=2022.09.2` means “install dask >=2022.09.2 with the extra `distributed` active”